components/Dropdown/Dropdown.js

```diff
@@ -1,28 +1,27 @@
 class Dropdown {
   constructor(element) {
-    
     // Assign this.element to the dropdown element
-    this.element;
-    
+    this.element = element;
+
     // Get the element with the ".dropdown-button" class found in the dropdown element (look at the HTML for context)
-    this.button = this.element.querySelector();
-    
+    this.button = this.element.querySelector(".dropdown-button");
+
     // assign the reference to the ".dropdown-content" class found in the dropdown element
-    this.content;
-    
-    // Add a click handler to the button reference and call the toggleContent method.
-    this.button.addEventListener('click', () => {
+    this.content = document.querySelector(".dropdown-content");
```

Comment:
this works fine, but for re-usability you'll want to select the content off of the element instead of the document.